test/state/timer-state.specs.js

```diff
@@ -370,9 +372,18 @@ describe('timer-state', () => {
     })
   })
 
+  describe('when setting shuffle mobbers on startup', () => {
+    beforeEach(() => timerState.setShuffleMobbersOnStartup(true))
+
+    it('should publish a configUpdated event', () => {
+      var event = assertEvent('configUpdated')
+      assert.deepStrictEqual(event.data.shuffleMobbersOnStartup, true)
+    })
+  })
+
   describe('getState', () => {
     describe('when getting non-default state', () => {
-      before(() => {
+      beforeEach(() => {

```

Comment:
I changed this from `before` to `beforeEach` because that impacts the order that things run in. I was experiencing a problem where the test was passing even without calling `setShuffleMobbersOnStartup` which this fixed.